Controllers/TodoController.cs

```diff
@@ -0,0 +1,48 @@
+using TodoNET6.Models;
+using TodoNET6.Services;
+
+namespace TodoNET6.Controllers
+{
+    public static class TodoController
+    {
+        public static async Task<IResult> GetTodos(
```

Comment:
Do you really have to say both `async` and `Task`? I can't be too surprised, but that feels a bit redundant.

Reply:
Yes, you do.

`Task` is essentially a `Promise` in C# everything marked async MUST return `Task<T>` or `Task`. You *can* do `async void` technically but that leads to thread locking issues in most cases.

Keep in mind that as you see C# code you will see `async Task` functions that do not appear to actually return a `Task` object. This is just syntactic sugar. The C# runtime intelligently and silently wraps your return values in a `Task` for you.